model/src/main/kotlin/config/CliConfiguration.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
```

Comment:
Should this only be 2020?

Reply:
Obsolete, because I removed that commit.